flatdata-rs/lib/src/vector.rs

```diff
@@ -121,83 +97,67 @@ where
     /// if capacity is already sufficient.
     #[inline]
     pub fn reserve(&mut self, additional: usize) {
-        let additional_bytes = Self::calc_size(self.len() + additional) - self.size_in_bytes();
-        self.data.reserve(additional_bytes)
-    }
-
-    /// Returns an `ArrayView` to this vector.
-    #[inline]
-    pub fn as_view(&self) -> ArrayView<T> {
-        ArrayView::new(&self.data[..self.size_in_bytes()])
+        self.data.reserve(self.data.len() + additional)
     }
 
-    /// Returns the contents of this vector as slice of bytes.
+    /// Returns a slice for the vector contents. Hides sentinels in case T IS_OVERLAPPING_WITH_NEXT
```

Comment:
Could you please add an empty line before `Hides ...`?

Reply:
Done